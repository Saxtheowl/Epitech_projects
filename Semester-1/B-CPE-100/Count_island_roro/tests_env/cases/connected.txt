XXX...
X.X...
XXX...
......
..XX..
..XX..