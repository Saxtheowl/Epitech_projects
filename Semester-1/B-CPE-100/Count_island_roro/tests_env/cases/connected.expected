000...
0.0...
000...
......
..11..
..11..
2